Zend/zend_ast.c

```diff
@@ -1842,6 +1842,11 @@ static ZEND_COLD void zend_ast_export_ex(smart_str *str, zend_ast *ast, int prio
 		case ZEND_AST_MATCH_ARM_LIST:
 			zend_ast_export_list(str, (zend_ast_list*)ast, 0, 0, indent);
 			break;
+		case ZEND_AST_PROPERTY_INITIALIZER_LIST:
```

Comment:
Does this need to have a case for ZEND_AST_INITIALIZER_EXPR? I'd assume it would, for propname: (expr)

This would affect php assert()

Reply:
Very nice catch! I also had to add code for `ZEND_AST_CLONE` in order to fix the behavior. I'll push it later, together with other fixes.